python-runtime/ftl/src/ftl/protos/xyz/block/ftl/v1/language/language_pb2.pyi

```diff
@@ -0,0 +1,273 @@
+from google.protobuf import struct_pb2 as _struct_pb2
```

Comment:
Mind adding these to the linguist generated file types?

Reply:
yee @alecthomas. done https://github.com/TBD54566975/ftl/pull/3301/commits/7e77dfc9979beedd5e12fbb4fc404924a8889e33. that look right to you?

Reply:
@alecthomas  so, i was curious about whether the line i added to `.gitattributes` actually worked. ran `github-linguist` in a docker container. turns out linguist automatically ignores generated protos!

```
find python-runtime/ftl/src/ftl/protos -type f -name "*.py" | head -n 1 | xargs github-linguist
python-runtime/ftl/src/ftl/protos/xyz/block/ftl/v1/ftl_pb2.py: 43 lines (37 sloc)
  type:      Text
  mime type: application/x-python
  language:  Python
  appears to be generated source code
```

there's several rules baked right into linguist which you can see [here](https://github.com/github-linguist/linguist/blob/main/lib/linguist/generated.rb#L91-L93)

<img width="1150" alt="image" src="https://github.com/user-attachments/assets/b7cf31b8-946e-42ba-8623-ee90f81fe75b">

Reply:
Yeah I was specifically referring to the pyi files

Reply:
aye, automatically identified as well:

```
python-runtime/ftl/src/ftl/protos/xyz/block/ftl/v1/controller_pb2.pyi: 348 lines (313 sloc)
  type:      Text
  mime type: text/plain
  language:  Python
  appears to be generated source code
```